docs/cookbook/adding-ajax-load-more-button-into-pagination.md

```diff
@@ -0,0 +1,140 @@
+# Adding Ajax Load More Button into Pagination
+
+This tutorial will guide us via steps needed for implementing page loader that will make calls to `Brand` controller. 
```

Comment:
I would put it this way: "In this cookbook, we will add a paginated brand list including ajax "load more" button to a product list page. After finishing the guide, you will know how to use multiple paginations on one page."